Sources/iOSClientExposure/Entitlements/Request/PlayEnigmaAsset.swift

```diff
@@ -61,6 +68,22 @@ public struct PlayEnigmaAsset: ExposureType {
         if let materialProfile = materialProfile {
             parameters["materialProfile"] = materialProfile
         }
+        
```

Comment:
Just a thought, most code in this struct is common with PlayEnigmaAdsAsset, so a lot of repetition could be avoided, especially the "parameters" part is identical in two places.

Is there a specific reason why struct is used and not class, considering the latter would have been helpful with inheritance / code sharing in this case.